images/ose-gcp-cloud-controller-manager.yml

```diff
@@ -1,7 +1,6 @@
 arches:
 - x86_64
 - ppc64le
-- aarch64
```

Comment:
Please drop #2125 from this PR, it was intended only for 4.12.

Reply:
Done